R/tte_source.R

```diff
@@ -4,17 +4,32 @@
 #'
 #' @details
 #' To see the definition of the various objects simply print the object in the
-#' R console, e.g. `print(death_event)`.
+#' R console, e.g. `print(death_event)`. For details of how to use these objects
+#' please refer to [admiral::derive_param_tte()].
+#'
+#' Printing an object will display input dataset_name, filter (if applicable), date variable,
+#' and appropriate values for EVNTDESC, CNSDTDSC, SRCDOM, SRCVAR, and SRCSEQ.
```

Comment:
Add `` around the var names, i.e. `EVNTDESC`, `CNSDTDSC`, `SRCDOM`, `SRCVAR`, and `SRCSEQ`.